src/tyr/serializers.cc

```diff
@@ -102,6 +102,12 @@ std::string serializeStatus(Api& request) {
   status_doc.AddMember("tileset_last_modified",
                        rapidjson::Value().SetInt(request.status().tileset_last_modified()), alloc);
 
+  rapidjson::Value actions_list(rapidjson::kArrayType);
+  for (const auto& action : request.status().available_actions()) {
+    actions_list.GetArray().PushBack(rapidjson::Value{}.SetString(action.c_str(), alloc), alloc);
+  }
+  status_doc.AddMember("available_actions", actions_list, alloc);
```

Comment:
would've like to switch this whole serializer to the managed `writer` syntax, so much more readable. but the `bbox` kinda prevents that, or we'd have to live with stringified coord floats. or write a geojson abstraction for the writer type..